runner/src/server/services/upload/uploadService.ts

```diff
@@ -60,7 +61,7 @@ export class UploadService {
       .join(", ");
     const lastAcceptedTypeName = acceptedTypeNames.slice(-1);
 
-    return `The selected file for "%s" must be a ${acceptedTypesNameWithoutLast} or ${lastAcceptedTypeName}`;
+    return `The selected files must be a ${acceptedTypesNameWithoutLast} or ${lastAcceptedTypeName}`;
```

Comment:
Correct me if I'm wrong, but if memory serves these errors are displayed as a group at the top of the page. If you have multiple upload fields, or fields in general, these errors should help the user identify which field has the error, so a name is useful?